src/main/java/net/openhft/chronicle/threads/MediumEventLoop.java

```diff
@@ -196,6 +196,9 @@ public void addHandler(@NotNull final EventHandler handler) {
         }
         do {
             pauser.unpause();
+            if (isStopped()) {
+                return;
+            }
```

Comment:
This is probably a change in behaviour, but if someone's adding a handler when the loop gets stopped it's probably fine to just return because that handler wouldn't have been executed anyway

Reply:
cool